RELEASE_NOTES.md

```diff
@@ -9,20 +9,28 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## New
-- Address Lookup functionality for Card Component. You can enable this feature by setting your address configuration to lookup while building your card configuration as follows
-```kotlin
-CheckoutConfiguration(
-    shopperLocale = shopperLocale,
-    environment = environment,
-    clientKey = clientKey,
-    amount = amount,
-    analyticsConfiguration = createAnalyticsConfiguration(),
-) {
-    card {
-        setAddressConfiguration(AddressConfiguration.Lookup())
+- Address Lookup functionality for Card Component. 
+  - You can enable this feature by setting your address configuration to lookup while building your card configuration as follows:
+    ```kotlin
+    CheckoutConfiguration(
+        shopperLocale = shopperLocale,
```

Comment:
WDYT about only writing the required parameters when initializing `CheckoutConfiguration`? 
```kotlin
    CheckoutConfiguration(environment, clientKey) {
        card {
            setAddressConfiguration(AddressConfiguration.Lookup())
        }
    }
```
Makes it a bit more readable